packages/flet_geolocator/CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# 0.22.0
+
+No changes in this release. Version bumped to follow parent `flet` package.
+
+# 0.21.2
+
+No changes in this release. Version bumped to follow parent `flet` package.
+
+# 0.21.1
+
+No changes in this release. Version bumped to follow parent `flet` package.
+
+# 0.21.0
+
+No changes in this release. Version bumped to follow parent `flet` package.
+
+# 0.20.2
+
+No changes in this release. Version bumped to follow parent `flet` package.
+
+# 0.20.1
+
+No changes in this release. Version bumped to follow parent `flet` package.
+
+# 0.20.0
+
+Initial release of the package.
```

Comment:
Please cleanup here (remove all the useless versions/descriptions) leaving only the below suggestion:
```suggestion
# 0.23.0

Initial release of the package.
```

Reply:
done